packit/copr_helper.py

```diff
@@ -242,7 +242,8 @@ def create_copr_project_if_not_exists(
                             f"when using Packit CLI."
                         )
                 raise PackitCoprSettingsException(
-                    f"Copr project update failed for '{owner}/{project}' project.",
+                    f"Copr project update failed for '{owner}/{project}' project."
+                    "because the Copr token is missing, invalid or expired",
```

Comment:
I am not sure if we can be sure that only the token is wrong. (E.g. the lack of permissions of the user itself is not an issue of the wrong token.)